marathon/dev-jupyter-cni-notebook-marathon.json

```diff
@@ -8,7 +8,7 @@
   "container": {
     "type": "MESOS",
     "docker": {
-      "image": "dcoslabs/dcos-jupyter:latest",
+      "image": "dcoslabs/dcos-jupyterlab:latest",
```

Comment:
Just a question of consistency: some of the docker images (e.g., in the dockerfiles above) are in the mesosphere org, while other in the dcoslabs. IMO if this is a beta product it should be in the mesosphere org.